CacheMoney/backend/pom.xml

```diff
@@ -20,20 +20,30 @@
 	<name>Cache Money</name>
 	<description>project3</description>
 
-
 	<properties>
 		<java.version>8</java.version>
 	</properties>
 
-
 	<dependencies>
 
+		<dependency>
+			<groupId>com.sun.mail</groupId>
+			<artifactId>javax.mail</artifactId>
+			<version>1.6.2</version>
+		</dependency>
+
 		<dependency>
 			<groupId>com.auth0</groupId>
 			<artifactId>java-jwt</artifactId>
 			<version>3.18.3</version>
 		</dependency>
 
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<version>2.1.210</version>
+		</dependency>
+
```

Comment:
Is there a way in maven to make this a developer dependency?

Reply:
Not that I see.